### PR TITLE
Renames various reagent IDs to be more descriptive

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
code: Renamed the IDs of various reagents to be more descriptive.
spellcheck: Fixed the descriptions of changeling adrenaling reagents.
/:cl:

I changed the reagent IDs of delayed chloral hydrate, KO beer and ling adrenals to be more descriptive. The outward appearance of KO beer and delayed chloral aren't affected by this.

- beer2 -> fakebeer
- changelingAdrenaline2 -> changelingHaste
- chloralhydrate2 -> chloralhydratedelayed

### CODE REVIEW

**Comment by DaedalusGame** [comment]
changlingAdrenaline2

**Comment by 81Denton** [comment]
when will it END
give me a minute

**Comment by ExcessiveUseOfCobblestone** [comment]
My only concern is that the name of the new chloral will be an obvious tip-off by someone with a medical analyzer that someone is running around with a sleepypen (assuming you only get to them after the mutetox and tirezine wear off).

**Comment by 81Denton** [comment]
@ExcessiveUseOfCobblestone That's actually a good reason to keep the name as it is. I didn't consider that you could metagame that.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1161,26 +1161,26 @@
 	. = 1
 	..()
 
-/datum/reagent/medicine/changelingAdrenaline2
-	name = "Adrenaline"
-	id = "changelingAdrenaline2"
-	description = "Drastically increases movement speed."
+/datum/reagent/medicine/changelingHaste
+	name = "Changeling Haste"
+	id = "changelingHaste"
```
Comment (DaedalusGame) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: you can't have it both ways, make this all lowercase

**Comment by ExcessiveUseOfCobblestone** [comment]
sCared?

**Comment by RandomMarine** [comment]
All ingame sources (Aside from the admin add reagent menu) should be changed to display the reagent's name instead of the id anyway.

**Comment by DaedalusGame** [comment]
they are

**Comment by ExcessiveUseOfCobblestone** [comment]
The only one that doesn’t to my knowledge  is the macros

**Comment by 81Denton** [comment]
The SCP coffee machine uses reagent IDs instead of names too

**Comment by MoreRobustThanYou** [comment]
@81Denton it uses both

**Comment by 81Denton** [comment]
Darn, I did not know that.

**Comment by MoreRobustThanYou** [comment]
yeah it checks for reagent ids first (it removes all spaces and lowercases the name). If it doesn't find one, then it checks all the reagent names

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**